### PR TITLE
HT-3183: Add dry-run flag for backup expiration

### PR DESCRIPTION
**Pull request by aelkiss**
This is so we can verify what it would delete in production without actually deleting anything.

### CODE REVIEW

**Comment by aelkiss** [comment]
Also fixed a couple of includes. I don't love sprinkling the concrete storage class names as requires in multiple places; maybe it makes sense just to require them from `HTFeed::Storage`?

File: lib/HTFeed/BackupExpiration.pm [hunk 1]
```diff
@@ -43,18 +50,27 @@ sub run {
     HAVING COUNT(*) > 1
 SQL
 
+  my $versions_sth = get_dbh()->prepare(<<'SQL');
```
Comment (aelkiss) [inline lib/HTFeed/BackupExpiration.pm:55]: Small optimization, but I noticed we were re-preparing the statements for every object, and it didn't seem like we needed to do that. @moseshll I did want to check there wasn't any particular reason for that though.

**Review by moseshll — APPROVED** [review]
The only thing that I find questionable is something I failed to implement: a sanity check on  `defined $storage_config` early in the process, maybe as part of the constructor. As far as what's actually here in this PR, runs fine for me. (I should elaborate: if you pass a bogus storage name on the CLI the script fails silently.)

File: lib/HTFeed/BackupExpiration.pm (hunk 1 above)
Comment (moseshll) [reply lib/HTFeed/BackupExpiration.pm:55]: No, my experience with the MySQL DBI in Perl has been to sidestep statement handles, so I don't tend to use them in particularly elegant ways.

**Comment by aelkiss** [comment]
Thanks for the quick turnaround. I'll add the check for `defined $storage_config`.

**Pull request merged**